### PR TITLE
[Fortaleza-2020] Event date change

### PR DESCRIPTION
**Pull request by renat473**
update date event

### CODE REVIEW

**Comment by mattstratton** [comment]
The commits on this PR are slightly confusing. 

I don't see any file changes in either commit that tell me the date changed?

**Comment by mattstratton** [comment]
I'm very confused about the history in this PR.

**Comment by renat473** [comment]
> I'm very confused about the history in this PR.

I'm checking what happened to PR

**Comment by MikeRosTX** [comment]
> I'm very confused about the history in this PR.

He's still working out the kinks. Chatting with Renato on Slack.

**Comment by mattstratton** [comment]
>He's still working out the kinks. Chatting with Renato on Slack.

Sounds good!

File: data/events/2020-fortaleza.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it
 # Note: we allow 2019-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
 
 # All dates are in unquoted 2019-MM-DD, like this:   variable: 2016-01-05
-startdate: 2020-09-26T08:00:00-03:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate: 2020-09-26T19:00:00-03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+startdate: 2020-08-03T08:00:00-03:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2020-08-03T18:00:00-03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (MikeRosTX) [inline data/events/2020-fortaleza.yml:15]: If you want October and mentioned in Slack, the dates should be:

**startdate: 2020-10-03
enddate: 2020-10-03**

**Comment by MikeRosTX** [comment]
@renat473 - Please confirm the dates. Because if it is in October, then the date needs to be changed.

**Comment by MikeRosTX** [comment]
Qual é o dia e o mês precisos do evento?

**Comment by MikeRosTX** [comment]
Confirmed actual data is **_03 de outubro de 2020_**

**Pull request merged**